### PR TITLE
feat: make rss great again

### PR DESCRIPTION
**Pull request by ArchiFleKs**

### CODE REVIEW

File: config.toml [hunk 1]
```diff
@@ -75,7 +75,7 @@ unsafe= true
     [Languages.fr]
     weight = 1
     languageCode = "fr-fr"
-    title = "Particule | Cloud Native Infrastructure"
+    title = "particule. | Cloud Native | GitOps | Infra as Code"
```
Comment (rguichard) [inline config.toml]: Not sure about than change.
Anyway, I'd prefer `particule. | Cloud Native - GitOps - Infra as Code`

**Review by rguichard — COMMENTED** [review]
About the RSS link, i'd prefer having it elsewhere. Maybe in the blog index page.

File: config.toml (hunk 1 above)
Comment (ArchiFleKs) [reply config.toml]: Yes I don't mind the `|` or  `-`

**Review by rguichard — APPROVED** [review]
lgtm

**Pull request merged**